README.rst

```diff
@@ -10,12 +10,12 @@ Installation
 
 The simplest way to install awscli-login plugin is to use pip::
 
-   $ pip install awscli-login
+    % pip install awscli-login
 
 After the awscli-login has been installed run the following command
 to enable the plugin::
 
-	$ aws configure set plugins.login awscli_login
```

Comment:
The spacing looks like it might be off here.

Reply:
Looks fine from here.